Sources/PodToBUILD/RuleUtils.swift

```diff
@@ -65,6 +65,18 @@ public func getPodBaseDir() -> String {
     return options.vendorize ? "Vendor" : "external"
 }
 
+/// We need to hardcode a copt to the $(GENDIR) for simplicity.
```

Comment:
Do we need to add this to our wiki instructions for people adopting this?

Reply:
We should add this to the `README.md`:  by default it isn't needed but for is flipped on headermaps. Might be useful since there are a lot of project generators